cdhweb/pages/templates/cdhpages/search.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'base.html' %}
+{% load humanize cdh_tags wagtailcore_tags %}
+{# Site search page #}
+
+{% block main %}
+<h1 class="sr-only">{{ page_title }}</h1>
+<form class="search" action="{% url 'search' %}" method="get">
+    {{ form.q }}
+    {{ form.filter }}
+    <input type="submit" value="Search">
+</form>
+<div class="results">
+    {% if page_obj %}
+    <p class="total-results">{{ page_obj.paginator.count }} Total Result{{ page_obj.paginator.count|pluralize}}</p>
+    <ol start="{{ page_obj.start_index }}">
+    {% for page in page_obj %}
+    {% include "cdhpages/snippets/search_result.html" %}
+    {% endfor %}
+    </ol>
+    {% elif form.q.value %}
+    <p class="total-results">No results found.</p>
+    {% endif %}
+</div>
+{% if page_obj and form.q.value %}
+<div class="pagination">
+    <span class="step-links">
+        {% if page_obj.has_previous %}
+            <a href="?{% url_replace 'page' '1' %}">&laquo; first</a>
+            <a href="?{% url_replace 'page' page_obj.previous_page_number %}">&lsaquo; previous</a>
+        {% endif %}
+        <span class="current">
+            Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}.
+        </span>
+        {% if page_obj.has_next %}
+            <a href="?{% url_replace 'page' page_obj.next_page_number %}">next &rsaquo;</a>
+            <a href="?{% url_replace 'page' page_obj.paginator.num_pages %}">last &raquo;</a>
+        {% endif %}
+    </span>
+</div>
+{% endif %}
```

Comment:
Can't we use the existing pagination template snippet here (snippets/pagination.html)? It looks like you shouldn't have to change it to much — maybe just adapt your url replace logic from this version.

If we use the same snippet, then pagination will be consistent with what we already have and we can change it in one place when we're ready to revise it.

Reply:
yeah — I reimplemented here because originally we were going to update the pagination as part of this issue, but now that we've carved it off into a separate design issue i think it makes sense to revert to the default pagination and then change it later in one place.